### ISSUE TITLE
Add HaikuOS support for MIPS 64 bits CPU

### ISSUE THREAD

**Issue by thomasarmel** [scrabsha/expendable#70]
The project doesn't compile on my HaikuOS, running on MIPS 64

**Comment by scrabsha** [comment]
Are you sure the computer is turned on?

**Comment by thomasarmel** [comment]
Yes, but I also recompiled the kernel and the libC to allocate all the variables on the Ethereum blockchain: https://github.com/alandefreitas/blockchain .
Maybe that's where the problem lies.

I followed advice from this video: https://www.youtube.com/watch?v=I6SRuvjIUls which told me Ethereum will soon rule the world.

**Comment by scrabsha** [comment]
I see. The issue is probably that there are no variable available on your device. I suggest adding more variable to your device by downloading more RAM. I can recommend https://downloadmoreram.com/, which really helped me in the past.

An alternative would be to use a lighter OS, for instance one that is coded in HTML.

**Comment by thomasarmel** [comment]
Still doesn't work... Maybe you should rewrite the entire project in Rust?
